pkg/scaffold/controller/controller.go

```diff
@@ -245,7 +250,7 @@ func (r *Reconcile{{ .Resource.Kind }}) Reconcile(request reconcile.Request) (re
 	found := &appsv1.Deployment{}
 	err = r.Get(context.TODO(), types.NamespacedName{Name: deploy.Name, Namespace: deploy.Namespace}, found)
 	if err != nil && errors.IsNotFound(err) {
-		log.Printf("Creating Deployment %s/%s\n", deploy.Namespace, deploy.Name)
+		log.Info(fmt.Sprintf("Creating Deployment %s/%s", deploy.Namespace, deploy.Name))
```

Comment:
For better or for worse, this is specifically discouraged by logr. https://github.com/go-logr/logr/blob/9fb12b3b21c5415d16ac18dc5cd42c1cfdd40c4e/logr.go#L11

logr wants you to use structured logging.

Reply:
You are right, I should follow the logr's pattern.
Thanks.